moped-editor/src/queries/project.js

```diff
@@ -288,26 +289,15 @@ export const TIMELINE_QUERY = gql`
   }
 `;
 
+// use this to update a single moped_proj_phase
 export const UPDATE_PROJECT_PHASES_MUTATION = gql`
   mutation ProjectPhasesMutation(
-    $phase_description: String
-    $is_current_phase: Boolean
-    $phase_start: date = null
-    $phase_end: date = null
     $project_phase_id: Int!
-    $phase_id: Int
-    $subphase_id: Int = 0
+    $object: moped_proj_phases_set_input!
```

Comment:
if folks don't have a taste for these generic object inputs, i don't mind changing it back to be explicit...